torch/_inductor/cpu_vec_isa.py

```diff
@@ -306,9 +320,24 @@ def _check_and_append_supported_isa(
 
     return supported_isa
 
+@functools.lru_cache(maxsize=None)
+def _is_arm_neoverse_v1() -> bool:
+    # reference: https://github.com/ARM-software/ComputeLibrary/blob/main/src/common/cpuinfo/CpuModel.cpp
+    try:
+        with open("/proc/cpuinfo") as _cpuinfo:
+            line = _cpuinfo.readline()
+            while line:
+                is_v1 = re.match(r"^CPU\spart(.*):\s0xd40(\n)$", line)
```

Comment:
Hi @aditew01 
Out of curiosity, is detection of Neoverse V1 the best way to detect SVE on any architecture? Wouldn't it be better to check for SVE and SVE2 directly from the CPUinfo?

Reply:
Hi @abhishek-iitmadras , Thanks for the comment. Yes, I too believe that's the best way as well. I was waiting for the cpuinfo PR + the SVE pr to get in and will start on refactoring this next.

Reply:
@abhishek-iitmadras revisiting this,
Since some logic are tied to the SVE 256 vec length, I'm leaving this logic as is, when we make the changes (for 128/ 512 vec length) we can revisit this. 

Ref code specific for SVE256: 
https://github.com/pytorch/pytorch/blob/eb359981df2c8f415b133380c92f392c98c5f356/aten/src/ATen/cpu/vec/functional_base.h#L113

Reply:
Yes. Suggest to use the cpuinfo library instead of `/proc/cpuinfo` which is Linux specific.

Reply:
Thanks for the suggestion @jgong5 . This totally makes sense, I've updated the logic, please check.